src/lib/crypto/kimchi_bindings/stubs/src/pasta_fq_plonk_index.rs

```diff
@@ -136,7 +136,7 @@ pub fn caml_pasta_fq_plonk_index_read(
     t.cs.fr_sponge_params = oracle::pasta::fq_3::params();
     t.srs = srs.clone();
     t.fq_sponge_params = oracle::pasta::fp_3::params();
-    t.linearization = expr_linearization(t.cs.domain.d1, false, None);
+    t.linearization = expr_linearization(t.cs.domain.d1, false, &None);
```

Comment:
this `&None` troubles me o_O, I don't understand why it needs to be a reference

Reply:
I'm still wondering how this compiles, since it is not expecting a reference to an option:

```rust
pub fn expr_linearization<F: FftField + SquareRootField>(
    domain: D<F>,
    chacha: bool,
    dummy_lookup_value: Option<&[F]>,
```

Reply:
ohh, it was changed to:

```rust
pub fn expr_linearization<F: FftField + SquareRootField>(
    domain: D<F>,
    chacha: bool,
    lookup_constraint_system: &Option<LookupConstraintSystem<F>>,
```